sdcm/loader.py

```diff
@@ -16,16 +16,21 @@
 
 from sdcm.utils import FileFollowerThread
 
+METRICS = {}
```

Comment:
why not global of the `CassandraStressExporter` class?

Reply:
yes, we can move it into the class. will do